### PR TITLE
fix(formative): rewrite and theme much more

### PR DESCRIPTION
**Pull request by r58Playz**
## 🔧 What does this fix? 🔧
Rewrote the theme, makes it a bit cleaner to read. Also fixed some bugs with the question elements.

## 🗒 Checklist 🗒

- [x] I have read and followed Catppuccin's [contributing guidelines](https://github.com/catppuccin/userstyles/blob/main/docs/CONTRIBUTING.md).
- [x] I have updated the version appropriately in the `==UserStyle==` header of the `catppuccin.user.css` file.

### CODE REVIEW

File: styles/formative/catppuccin.user.css [hunk 1]
```diff
@@ -54,10 +50,14 @@
     @mantle: @catppuccin[@@lookup][@mantle];
     @crust: @catppuccin[@@lookup][@crust];
     @accent-color: @catppuccin[@@lookup][@@accent];
-    @contrast-accent-color: @catppuccin[@@lookup][@base];
+    & when (@lookup = latte) {
+      --contrast-accent-color: @catppuccin[@@lookup][@text];
+    }
+    & when not (@lookup = latte) {
+      --contrast-accent-color: @catppuccin[@@lookup][@base];
+    }
```
Comment (uncenter) [inline styles/formative/catppuccin.user.css:58]:
```suggestion
    --contrast-accent-color: if(@lookup=latte, @text, @base);
```

File: styles/formative/catppuccin.user.css [hunk 2]
```diff
@@ -81,1054 +81,665 @@
...
-      color: @contrast-accent-color !important;
-      span {
-        color: @contrast-accent-color;
-      }
-    }
-    div.ckhGGP {
-      background-color: @base;
-    }
+    width: 100%;
+    height: 100%;
```
Comment (uncenter) [inline styles/formative/catppuccin.user.css:85]:
```suggestion
```

Avoid setting non-color properties if you don't have to.

File: styles/formative/catppuccin.user.css [hunk 3]
```diff
@@ -81,1054 +81,665 @@
...
-          }
         }
       }
-    }
-    .iCrZor {
-      &.selectedTab {
-        background-color: @accent-color;
-        color: @contrast-accent-color;
+      .ChoiceElement__RootDiv-sc-1mbtxfk-1 {
+        border-radius: 4px;
```
Comment (uncenter) [inline styles/formative/catppuccin.user.css:571]:
```suggestion
```

See prior comment about non-color properties.

File: styles/formative/catppuccin.user.css [hunk 4]
```diff
@@ -81,1054 +81,665 @@
...
+      .DrawingCardButton__RootButton-sc-u2qprg-1 {
+        border-color: @overlay0;
+        .DrawingCardButton__DrawingPreview-sc-u2qprg-3 {
+          background: rgb(245, 246, 248);
         }
-        .JoinQuickCode__ErrorDiv-sc-1uhfett-4 {
-          color: @red;
+        .DrawingCardButton__HoverDarkScrim-sc-u2qprg-2 {
+          background: fade(@crust, 75%);
+          border-radius: 0;
```
Comment (uncenter) [inline styles/formative/catppuccin.user.css:587]:
```suggestion
```

File: styles/formative/catppuccin.user.css [hunk 5]
```diff
@@ -81,1054 +81,665 @@
...
+        z-index: 2;
+      }
+      .Segment__SegmentSpan-sc-r0cy9o-0 {
+        background: @surface0 !important;
+      }
+      .TrimmerMarker__RootSpan-sc-1f17hi4-0 {
+        background: @overlay0;
+        z-index: 1;
+      }
+    }
```
Comment (uncenter) [inline styles/formative/catppuccin.user.css:655]:
```suggestion
    .MediaSlider__RootSpan-sc-1gz67jn-0 {
      .TrimmerMarker__TooltipDiv-sc-1f17hi4-1 {
        background: @surface2;
        color: @text;
      }
      .Thumb__RootSpan-sc-yv6w5k-0 {
        background: @accent-color;
      }
      .Segment__SegmentSpan-sc-r0cy9o-0 {
        background: @surface0 !important;
      }
      .TrimmerMarker__RootSpan-sc-1f17hi4-0 {
        background: @overlay0;
      }
    }
```

**Review by uncenter — CHANGES_REQUESTED** [review]

File: styles/formative/catppuccin.user.css (hunk 5 above)
Comment (r58Playz) [reply styles/formative/catppuccin.user.css:655]: The `z-index`s are needed to fix this issue in the player. Is it not allowed to do so?
image

File: styles/formative/catppuccin.user.css (hunk 2 above)
Comment (r58Playz) [reply styles/formative/catppuccin.user.css:85]: Ah, there was a white flash as the page loaded (which takes quite a while) so wanted to fix that.

File: styles/formative/catppuccin.user.css (hunk 4 above)
Comment (r58Playz) [reply styles/formative/catppuccin.user.css:587]: Border-radius here is used to fix this small white area in the drawing preview. Is it not allowed to do so?
image

**Comment by uncenter** [comment]
Alright I guess if the website is in a broken site like that you can use non-color properties.

**Review by isabelroses — APPROVED** [review]

**Pull request merged**